src/components/ArcanaSettingsTab.ts

```diff
@@ -57,6 +57,218 @@ export default class ArcanaSettingsTab extends PluginSettingTab {
           });
       });
 
+	new Setting(containerEl)
+      .setName('Input language')
```

Comment:
We can change to something more general when we get the feature to work in places other than TTS
```suggestion
      .setName('Text to Speech Language')
```

Reply:
agreed!